### ISSUE TITLE
Integrate work from fedora-mailing-list-analysis into this repo

### ISSUE THREAD

**Issue by MichaelClifford** [aicoe-aiops/mailing-list-analysis-toolkit#31]
**Is your feature request related to a problem? Please describe.**
We currently have two repos targeted at analyzing Fedora mailing list data.   

**Describe the solution you'd like**
Merge the two projects by incorporating `/fedora-mailing-list-analysis` work to date into this repo and continuing to work from here. 

**Describe alternatives you've considered**
We've considered maintaining two similar project repo's and determined that it was not a good idea.

**Comment by oindrillac** [comment]
Great! Thanks for creating this issue @MichaelClifford ,

I will share some points from my discussion with @cdolfi today. We have decided that :-
- We would be using this repo to add the analysis on hate speech and discriminatory work rather than the fedora mailing list analysis repo given that this is at a more polished stage and @cdolfi can leverage the clean data to start work on the machine learning and analysis bits.
- As a first step, to familiarize herself better with the repo, I have encouraged her to try out the notebooks in the repo and open any issues on the repo with any questions - related to access, code, just about anything which might be unclear. 
- Once familiar with the repo, she would be adding any cleaning scripts or code that she finds missing here that she has implemented, that could be an addition to the data ETL bits in this repo.
- Her work on hate speech detection could also be an addition to the analysis notebooks within the repo and she could leverage  the dashboard that is already created to present her work.
